.github/workflows/sync.yml

```
@@ -106,6 +106,12 @@ jobs:
             else
               echo "⚠️ Las ramas necesitan sincronización"
               echo "need_sync=true" >> $GITHUB_OUTPUT
+              
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)